frontend/views/containers/user-settings/AppLogs.vue

```diff
@@ -0,0 +1,177 @@
+<template lang='pug'>
+  .settings-container
+    section.card
+      .c-header
+        fieldset.c-filters
+          legend.sr-only Visible logs
+          label.checkbox
+            input.input(type='checkbox' name='filter' v-model='form.filter' value='error')
+            i18n Errors
+          label.checkbox
+            input.input(type='checkbox' name='filter' v-model='form.filter' value='warn')
+            i18n Warnings
```

Comment:
Here, show but disable these two checkboxes. Make sure that they're always on and cannot be disabled.

Reply:
Hum... If they are disabled, why should we show them? Maybe we shouldn't even have filters? In what scenario would the user wanna (un)check the debug/info/log filters?

Also, we don't have the UI for disabled checkboxs.

cc @mmbotelho

Reply:
> Hum... If they are disabled, why should we show them?

To explain to the user why they're still seeing log messages even when they've unchecked everything. So that users understand what's going on.

> Maybe we shouldn't even have filters?

We definitely need them. We do not want to log `debug` or `log` by default in production.

Reply:
> Question: In what scenario would the user (un)check debug/info/log filters?

By default `debug` and `log` are unchecked in production. Users will enable these checkboxes if they run into problems and we ask them to enable them.